src/cpp/ext/otel/otel_server_call_tracer.cc

```diff
@@ -131,16 +134,17 @@ class OpenTelemetryServerCallTracer : public grpc_core::ServerCallTracer {
  private:
   absl::Time start_time_;
   absl::Duration elapsed_time_;
-  std::string method_;
+  grpc_core::Slice path_;
+  absl::string_view method_;
```

Comment:
Instead of storing this as a data member, we could just have a method that returns this value by calling `absl::StripPrefix(path_.as_string_view(), "/")`.  That would reduce per-call memory with very little additional CPU overhead.

Same thing in the client call tracer, I suspect.

Reply:
done